src/cljs/org/broadinstitute/firecloud_ui/page/profile.cljs

```diff
@@ -29,15 +29,49 @@
      (cond (:error-message @state) (style/create-server-error-message (:error-message @state))
            (:values @state)
            [:div {}
-            (react/call :render-field this :name "Full Name:")
-            (react/call :render-field this :email "Contact Email:")
-            (react/call :render-field this :institution "Institutional Affiliation:")
-            (react/call :render-field this :pi "Principal Investigator:")
+            [:div {:style {:fontWeight "bold" :margin "1em 0 1em 0"}} "* - required fields"]
+            (react/call :render-field this :firstName "*First Name:")
+            (react/call :render-field this :lastName "*Last Name")
+            (react/call :render-field this :title "*Title:")
+            (react/call :render-field this :institute "*Institute:")
+            (react/call :render-field this :institutionalProgram "*Institutional Program:")
+            [:div {:style {:fontSize "88%"}} "Program Location:"
+              [:div {}
+                (react/call :render-nested-field this :programLocationCity "*City:")
+                (react/call :render-nested-field this :programLocationState "*State/Province:")
+                (react/call :render-nested-field this :programLocationCountry "*Country:")]]
+                (react/call :render-field this :pi "*Principal Investigator/Program Lead:")
+            [:div {:style {:marginBottom "1em" :fontSize "88%"}} "*NonProfit Status:"
+              (react/call :render-radio-field this :nonProfitStatus "Profit")
+              (react/call :render-radio-field this :nonProfitStatus "Non-Profit")]
+            (react/call :render-field this :billingAccountName "Google Billing Account Name")
```

Comment:
I know we don't like camel-casing, but if I used dashes, the json comes through un-camel-cased and broken. Would like a approach to bridge that gap.